### DISCUSSION TITLE
Reports

### DISCUSSION THREAD

**Discussion by Goloka** [Eqonomize/Eqonomize#171] [General]
good day
 I'm recently using the system
 I wanted to know if it is possible to add any report
 Has the available option
 thank you so much
 greetings from Buenos Aires

**Comment by hanna-kn** [comment]
I suggest that you try to rephrase the question to make it clearer.

**Comment by Goloka** [comment]
hello,
 Is it possible to generate a report with some options that I don't see available?

 for example a list that leakes by movements with labels

 thanks for your help

**Comment by hanna-kn** [comment]
> Is it possible to generate a report with some options that I don't see available?

No.

> for example a list that leakes by movements with labels

Please use your native language instead (I will probably have a better chance of finding a reasonable translation if I run the text through a translator myself).

**Comment by Goloka** [comment]
buen dia
disculpas por mi mal ingles

la consulta es

se puede hacer un reporte que filtre movimientos por etiqueta?

**Comment by hanna-kn** [comment]
I'm not sure I understand what you mean with "movimientos".

The development over time report (reporte detallado por tiempo) can be filtered on one or multiple tags/etiquetas (select "Tags"/"Etiquetas" in the left-most source/fuente menu and selecting desired tags in the second menu), or show each tag as separate columns (for expenses or incomes).

The categories comparison report (reporte comparativo de categorías) can be filtered on a single tag (using the items at the end of the source menu), or show tag as separate rows (all tags source) or columns.

If you are asking for a way the filter transfers (transferencias) using tags (etiquetas), this can be done for the transfers list be entering a complete tag in the description field.
